catalyst_voices/pubspec.yaml

```diff
@@ -38,6 +38,8 @@ dependencies:
   flutter_adaptive_scaffold: ^0.1.11
   flutter_bloc: ^8.1.3
   flutter_localized_locales: ^2.0.5
+  flutter_quill: ^10.5.0
+  flutter_quill_extensions: ^10.5.0
```

Comment:
Pls add these versions in melos.yaml. Then they will be synced across all packages and we will avoid using different version of the same package.